content/en/docs/concepts/workloads/pods/pod-qos.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Pod Quality of Service Classes
+content_type: concept
+weight: 30
```

Comment:
```suggestion
weight: 85
```
We avoid duplicating any weight value - that makes it hard to localize things. Sorry for spotting this late.